responsive.css

```diff
@@ -0,0 +1,367 @@
+    width: 85%;
+  }
+
+  .article2-subelement {
+    margin-left: 25px;
+  }
+
+  .more-list {
+    margin: 25px 50px;
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 20px;
+  }
+
+  .more-list .header-img {
+    max-height: 200px;
+    width: 290px;
+  }
+
+  .hardfork-img {
+    position: relative;
+  }
+
+  .hardfork-img .hardfork-content {
+    position: absolute;
+    top: 125px;
+    display: flex;
+    flex-direction: column;
+    font-size: small;
+    padding-left: 10px;
+    color: white;
+  }
+}
+
+@media only screen and (max-width: 767px) {
+  .nav1,
+  .nav2,
+  .mobile-nav2 {
+    display: none;
+  }
+
+  #tnw-main-article-image {
+    display: none;
+  }
+
+  .mobile-img {
+    background: rgb(210, 18, 137);
+    background:
+      linear-gradient(
+        45deg,
+        rgba(210, 18, 137, 1) 0%,
+        rgba(255, 67, 34, 1) 100%
+      );
+    width: 80px;
+    padding-top: 20px;
+  }
+
+  .mobile-nav {
+    display: flex;
+    position: fixed;
+    width: 100%;
+    justify-content: space-between;
+    background-color: #fff;
+  }
+
+  .mobile-nav-search {
+    border-left: 1px solid rgb(180, 180, 180);
+    border-right: 1px solid rgb(180, 180, 180);
+    padding: 15px;
+    cursor: pointer;
+  }
+
+  .mobile-nav .fa-search:hover {
+    color: black;
+  }
+
+  .mobile-nav-more {
+    padding: 15px;
+  }
+
+  .mobile-nav1 {
+    display: flex;
+  }
+
+  .mobile-nav2 {
+    background: rgb(210, 18, 137);
+    background:
+      linear-gradient(
+        45deg,
+        rgba(210, 18, 137, 1) 0%,
+        rgba(255, 67, 34, 1) 100%
+      );
+  }
+
+  .mobile-nav2 img {
+    width: 66px;
+    margin-top: 20px;
+  }
+
+  section {
+    padding: 5px;
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-auto-rows: 200px;
+    grid-gap: 10px;
+    grid-template-areas:
+      "section1"
+      "section2"
+      "section3";
+  }
+
+  .section1 > * {
+    padding: 10px;
+    padding-top: 100px;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+
+  .section1,
+  .section2,
+  .section3 {
+    height: 150px;
+  }
+
+  .showcase-title {
+    font-size: 15px;
+    font-weight: 700;
+  }
+
+  .latest-news {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .news-pages {
+    margin: 25px 20px;
+  }
+
+  .news-container {
+    display: flex;
+    align-items: center;
+    width: 100%;
+  }
+
+  .news-container > img {
+    width: 75px;
+    height: 75px;
+    padding: 15px;
+  }
+
+  .article2 {
+    display: flex;
+    flex-direction: column;
+    margin-left: 50px;
+  }
+
+  .article2-subelement {
+    margin-left: 25px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .article-container {
+    display: flex;
+    flex-direction: column;
+    margin: 0 auto;
+    width: 85%;
+  }
+
+  .more-list {
+    margin: 25px 50px;
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+  }
+
+  .more-list .header-img {
+    width: 647px;
+  }
```

Comment:
- [ ] You have an horizontal scroll because your images get out of the screen, try to not use fixed width in px, and try to use % to avoid this problem
![image](https://user-images.githubusercontent.com/53256139/88814961-5f1d5b80-d180-11ea-9cc3-02b09b7c1047.png)